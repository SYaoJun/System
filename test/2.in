Name,Race,Age,Weapon
Frodo Baggins,Hobbit,51,Sting
Aragorn,Human,87,Anduril
Gandalf,Istari,Unknown,Staff
Samwise Gamgee,Hobbit,38,Cast iron pan