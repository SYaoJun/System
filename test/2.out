Name
Frodo Baggins
Aragorn
Gandalf
Samwise Gamgee
